db/migrate/20200115090742_fix_parent_id_for_wiki_pages.rb

```diff
@@ -0,0 +1,9 @@
+class FixParentIdForWikiPages < ActiveRecord::Migration[6.0]
+  def up
+    MyWikiPages.where(parent_id: 0).update_all(parent_id: nil)
+  end
+end
+
+class MyWikiPages < ActiveRecord::Base
```

Comment:
It's best to move this into the Migration class itself. As in

```ruby
class FixParentIdForWikiPages < ActiveRecord::Migration[6.0]
  class MyWikiPages < ActiveRecord::Base
...
```

Just to avoid possible name conflicts in the root namespace no matter how unlikely it is that someone else will call their class that. ;)

Reply:
👍